### PR TITLE
Used table format for README summary of Commands.

### PR DESCRIPTION
**Pull request by wayiam**
**Please ensure that your pull request fulfills these requirements:**
- [x] The pull request is being made against the `master` branch
- [x] Tests for the changes have been added (for bug fixes / features)

**What is the purpose of this pull request? (bug fix, enhancement, new feature,...)**

**What changes did you make?**

**Provide some example code that this change will affect, if applicable:**

**Is there anything you'd like reviewers to focus on?**

**Please provide testing instructions, if applicable:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,53 +42,32 @@ Using `npx` you can run the script without installing it first:
...
-
-`--no-dotfiles` Do not show dotfiles
-
-`--mimetypes` Path to a .types file for custom mimetype definition
-
-`-h` or `--help` Print this list and exit.
-
-`-v` or `--version` Print the version and exit.
+| Command         | 	Description         |
+| -------------    |:-------------:|
```
Comment (thornjad) [inline README.md:46]: Would left-aligned text be easier to read?

**Review by thornjad — CHANGES_REQUESTED** [review]
This is looking good! What do you think about adding a third column to display default values?

File: README.md (hunk 1 above)
Comment (wayiam) [reply README.md:46]: > This is looking good! What do you think about adding a third column to display default values?

Definitely, third column for the Default values is a good idea. But not for all the command Description have their default values mentioned. What should i do in that case?

File: README.md [hunk 2]
```diff
@@ -42,53 +42,32 @@ Using `npx` you can run the script without installing it first:
...
-
-`-h` or `--help` Print this list and exit.
-
-`-v` or `--version` Print the version and exit.
+| Command         | 	Description         | Defaults  |
+| -------------  |-------------|-------------|
+|`-p` or `--port` |Port to use. Use `-p 0` to look for an open port, starting at 8080. It will also read from `process.env.PORT`. |8080 |
+|`-a`   |Address to use |0.0.0.0|
+|`-d`     |Show directory listings |`true` |
+|`-i`   |Address to use | 0.0.0.0 |
```
Comment (thornjad) [inline README.md:50]:
```suggestion
|`-i`   | Display autoIndex | `true` |
```

File: README.md [hunk 3]
```diff
@@ -42,53 +42,32 @@ Using `npx` you can run the script without installing it first:
...
+|`-g` or `--gzip` |When enabled it will serve `./public/some-file.js.gz` in place of `./public/some-file.js` when a gzipped version of the file exists and the request accepts gzip encoding. If brotli is also enabled, it will try to serve brotli first.|`false`|
+|`-b` or `--brotli`|When enabled it will serve `./public/some-file.js.br` in place of `./public/some-file.js` when a brotli compressed version of the file exists and the request accepts `br` encoding. If gzip is also enabled, it will try to serve brotli first. |`false`|
+|`-e` or `--ext`  |Default file extension if none supplied |`html` | 
+|`-s` or `--silent` |Suppress log messages from output  | |
+|`--cors` |Enable CORS via the `Access-Control-Allow-Origin` header  | |
+|`-o [path]` |Open browser window after starting the server. Optionally provide a URL path to open. e.g.: -o /other/dir/ | |
+|`-c` |Set cache time (in seconds) for cache-control max-age header, e.g. `-c10` for 10 seconds. To disable caching, use `-c-1`.|`false` |
```
Comment (thornjad) [inline README.md:57]:
```suggestion
|`-c` |Set cache time (in seconds) for cache-control max-age header, e.g. `-c10` for 10 seconds. To disable caching, use `-c-1`.|`3600` |
```

File: README.md [hunk 4]
```diff
@@ -42,53 +42,32 @@ Using `npx` you can run the script without installing it first:
...
+|`-U` or `--utc` |Use UTC time format in log messages.| |
+|`--log-ip` |Enable logging of the client's IP address |`false` |
+|`-P` or `--proxy` |Proxies all requests which can't be resolved locally to the given url. e.g.: -P http://someurl.com | |
+|`--username` |Username for basic authentication [none] | |
+|`--password` |Password for basic authentication [none] | |
```
Comment (thornjad) [inline README.md:62]:
```suggestion
|`--username` |Username for basic authentication | |
|`--password` |Password for basic authentication | |
```

File: README.md [hunk 5]
```diff
@@ -42,53 +42,32 @@ Using `npx` you can run the script without installing it first:
...
+|`-S` or `--ssl` |Enable https.| |
+|`-C` or `--cert` |Path to ssl cert file |`cert.pem` | 
+|`-K` or `--key` |Path to ssl key file |`key.pem` |
+|`-r` or `--robots` |Provide a /robots.txt `User-agent: *\nDisallow: /` | |
```
Comment (thornjad) [inline README.md:66]: This one's also an improvement to the existing description

```suggestion
|`-r` or `--robots` | Automatically provide a /robots.txt (The content of which defaults to `User-agent: *\nDisallow: /`)  | `false` |
```

**Review by thornjad — CHANGES_REQUESTED** [review]

**Review by thornjad — APPROVED** [review]

**Comment by wayiam** [comment]
Hey, is there anything left to do in my PR?

**Comment by thornjad** [comment]
Was just waiting for the tests to pass! Looks all good!

**Pull request merged**